submarine-server/server-core/src/test/java/org/apache/submarine/server/experiment/database/ExperimentServiceTest.java

```
@@ -0,0 +1,139 @@
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.submarine.server.experiment.database;
+
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ExperimentServiceTest {
+  private static final Logger LOG = LoggerFactory.getLogger(ExperimentServiceTest.class);
+  ExperimentService experimentService = new ExperimentService();
+
+  @After
+  public void cleanExperimentTable() throws Exception {
+    List<ExperimentEntity> entities = experimentService.selectAll();
+    for (ExperimentEntity entity: entities) {
+      experimentService.delete(entity.getId());
+    }
+  }
+
+  @Test
+  public void testInsert() throws Exception {
+    try {
+      ExperimentEntity entity = new ExperimentEntity();
+      String id = "experiment_1230";
+      String spec = "{\"value\": 1}";
+
+      entity.setId(id);
+      entity.setExperimentSpec(spec);
+
+      experimentService.insert(entity);
+
+      ExperimentEntity entitySelected = experimentService.select(id);
+
+      compareEntity(entity, entitySelected);
+    } catch (Exception e) {
+      LOG.error(e.getMessage(), e);
+      throw new Exception(e);
+    }
```

No need to try ... catch in a test. Just let it throw exceptions.